src/styles/utils/normalize.scss

```diff
@@ -0,0 +1,7 @@
+body,
+header,
+nav,
+ul {
+  box-sizing: border-box;
+  font-family: "Roboto", Helvetica, sans-serif;
```

Comment:
`font-family` can be inherited - you need to set it only for the upper-level tag

more on this and `box-size` inheritance see https://medium.com/code-writers/understanding-how-inheritance-works-in-css-35efb4f64fc5#:~:text=or%20non%2Dinherited.-,Inherited%20properties,font%2Dsize%20from%20its%20parent.